content/docs/quickstart/aws/destroy-stack.md

```diff
@@ -42,7 +42,9 @@ Resources:
 Duration: 26s
 ```
 
-To delete the stack itself, run `pulumi stack rm`.
+To delete the stack itself, run `pulumi stack rm`. Note that this removes the stack
+entirely from `pulumi.com`, along with all of its update history. See [pulumi stack
```

Comment:
Re: "entirely from `pulumi.com`". I assume we'll fix this up later to use whatever consistent naming we decide (e.g. "Pulumi Console" vs. "Pulumi Service").